flatm8s/src/pages/Home.js

```diff
@@ -4,16 +4,12 @@ import { useLocation } from 'react-router-dom'
 
 
 export default function Home() {
-
-
         const location = useLocation()
+        console.log(location.state);
```

Comment:
Do you still want this?

Reply:
just as a test to know that data has been passed correctly to Home